influxdb2_client/src/models/health.rs

```diff
@@ -0,0 +1,45 @@
+//! Health
+
+use serde::{Deserialize, Serialize};
+
+/// HealthCheck
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HealthCheck {
+    /// Name of the infludb instance
```

Comment:
Not sure if this typo came from the swagger definition or not, but we should fix it in either case:

```suggestion
    /// Name of the influxdb instance
```

Reply:
It was my mistake, I'll fix it

Reply:
I'll add integration test too!